### PR TITLE
cast functions and operators

### PR DESCRIPTION
**Pull request by XuHuaiyu**
cast 方法相关文档

### CODE REVIEW

**Comment by shenli** [comment]
文件名最好换成英文的，否则不好操作
Rest LGTM

File: functions-and-operators/Cast 函数和操作符.md [hunk 1]
```diff
@@ -0,0 +1,13 @@
+# Cast 函数和操作符
+
+**Cast 函数和操作符**
```
Comment (andelf) [inline functions-and-operators/Cast 函数和操作符.md]: use TOC

File: functions-and-operators/Cast 函数和操作符.md [hunk 2]
```diff
@@ -0,0 +1,13 @@
...
+
+| 函数名                                      | 功能描述               |
+| ---------------------------------------- | ------------------ |
+| [`BINARY`](https://dev.mysql.com/doc/refman/5.7/en/cast-functions.html#operator_binary) | 将一个字符串转换成一个二进制字符串。 |
+| [`CAST()`](https://dev.mysql.com/doc/refman/5.7/en/cast-functions.html#function_cast) | 将一个值转换成一个确定类型。     |
+| [`CONVERT()`](https://dev.mysql.com/doc/refman/5.7/en/cast-functions.html#function_convert) | 将一个值转换成一个确定类型。     |
+
+Cast 函数和操作符实现值从一个类型到另一个类型的转换。
```
Comment (andelf) [inline functions-and-operators/Cast 函数和操作符.md]: Use upper case for your function name

File: functions-and-operators/Cast 函数和操作符.md (hunk 1 above)
Comment (andelf) [reply functions-and-operators/Cast 函数和操作符.md]: use upper case

Comment (XuHuaiyu) [reply functions-and-operators/Cast 函数和操作符.md]: This `Cast` is not a function name,
but represents a kind of functions.

File: functions-and-operators/Cast 函数和操作符.md (hunk 2 above)
Comment (XuHuaiyu) [reply functions-and-operators/Cast 函数和操作符.md]: ditto

**Comment by XuHuaiyu** [comment]
PTAL @andelf

File: sql/cast-functions-and-operators.md [hunk 3]
```diff
@@ -0,0 +1,13 @@
+# Cast 函数和操作符
+
+## Cast 函数和操作符
```
Comment (XuHuaiyu) [inline sql/cast-functions-and-operators.md]: This is the name of the table,
which label should I use here? @andelf

File: sql/cast-functions-and-operators.md [hunk 4]
```diff
@@ -0,0 +1,13 @@
...
+
+| 函数名                                      | 功能描述               |
+| ---------------------------------------- | ------------------ |
+| [`BINARY`](https://dev.mysql.com/doc/refman/5.7/en/cast-functions.html#operator_binary) | 将一个字符串转换成一个二进制字符串。 |
+| [`CAST()`](https://dev.mysql.com/doc/refman/5.7/en/cast-functions.html#function_cast) | 将一个值转换成一个确定类型。     |
+| [`CONVERT()`](https://dev.mysql.com/doc/refman/5.7/en/cast-functions.html#function_convert) | 将一个值转换成一个确定类型。     |
+
+Cast 函数和操作符实现值从一个类型到另一个类型的转换。
```
Comment (queenypingcap) [inline sql/cast-functions-and-operators.md]: Move the following statement to the beginning of the doc.
Cast 函数和操作符用于将某种数据类型的值转换为另一种数据类型。TiDB 中该函数和操作符的使用方法与 MySQL基本一致，详情参见: [Cast Functions and Operators](https://dev.mysql.com/doc/refman/5.7/en/cast-functions.html).

File: sql/cast-functions-and-operators.md (hunk 3 above)
Comment (queenypingcap) [reply sql/cast-functions-and-operators.md]: Feel free not to use any label here.

File: sql/cast-functions-and-operators.md [hunk 5]
```diff
@@ -0,0 +1,13 @@
+# Cast 函数和操作符
```
Comment (queenypingcap) [inline sql/cast-functions-and-operators.md]: Please add the following metadata to the beginning of the .md file.
---
title: TiDB 用户文档
category: user guide
---

**Comment by XuHuaiyu** [comment]
PTAL @queenypingcap

**Comment by XuHuaiyu** [comment]
PTAL @andelf

File: sql/cast-functions-and-operators.md [hunk 6]
```diff
@@ -0,0 +1,19 @@
...
+
+
+# Cast 函数和操作符
+
+Cast 函数和操作符用于将某种数据类型的值转换为另一种数据类型。TiDB 中该函数和操作符的使用方法与 MySQL基本一致，详情参见: [Cast Functions and Operators](https://dev.mysql.com/doc/refman/5.7/en/cast-functions.html).
+
+**Cast 函数和操作符表**
+
+| 函数和操作符名                                  | 功能描述               |
+| ---------------------------------------- | ------------------ |
```
Comment (andelf) [inline sql/cast-functions-and-operators.md]: | -------------- | ----------------------------------------

**Comment by andelf** [comment]
minor changes required make this a LGTM. :)

And

Better use CAST instead of Cast for consistency with MySQL?

**Comment by XuHuaiyu** [comment]
MySQL use `CAST` only in the table
https://dev.mysql.com/doc/refman/5.7/en/cast-functions.html

**Pull request merged**